src/main/java/org/graylog/integrations/notifications/types/SlackEventNotificationConfig.java

```diff
@@ -1,16 +1,16 @@
 /**
  * This file is part of Graylog.
- *
+ * <p>
  * Graylog is free software: you can redistribute it and/or modify
  * it under the terms of the GNU General Public License as published by
  * the Free Software Foundation, either version 3 of the License, or
  * (at your option) any later version.
- *
+ * <p>
  * Graylog is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  * GNU General Public License for more details.
- *
+ * <p>
  * You should have received a copy of the GNU General Public License
  * along with Graylog.  If not, see <http://www.gnu.org/licenses/>.
  */
```

Comment:
Again wondering why the change to the license info

Reply:
hope, this is a minor issue and can be fixed with a simple mvn command.

Reply:
But why did you decide to change the license as part of this PR?

Reply:
i didn't intend to change. I noticed this, only after I raised the PR.